python/cudf_polars/cudf_polars/experimental/parallel.py

```
@@ -146,7 +122,10 @@ def evaluate_streaming(
     # Clear source info cache in case data was overwritten
     _clear_source_info_cache()
 
-    return evaluate_rapidsmpf(ir, config_options)
+    from cudf_polars.experimental.rapidsmpf.core import evaluate_logical_plan
```

Ruff (0.15.12)

[warning] `import` should be at the top-level of a file

(PLC0415)